pkg/internal/checkup/vmi/spec.go

```diff
@@ -82,6 +76,11 @@ func WithDataVolume(volumeName string, pvc *corev1.PersistentVolumeClaim, snap *
 				Namespace: snap.Namespace,
 				Name:      snap.Name,
 			}
+		} else {
```

Comment:
I think you want to pass options to `WithDataVolume`  similarly to https://github.com/kubevirt/kubevirt/blob/main/pkg/libvmi/cloudinit.go#L31

Reply:
Changed, but not sure how much I like it. wdyt?